package.json

```diff
@@ -8,11 +8,11 @@
     "lint": "jshint index.js lib/*.js"
   },
   "dependencies": {
-    "async": "^1.5.2",
+    "async": "^2.3.0",
     "confit": "^2.0.0",
-    "debug": "^2.1.0",
+    "debug": "^2.6.4",
     "lodash": "^4.12.0",
-    "selenium-webdriver": "^2.53.2",
+    "selenium-webdriver": "2.53.3",
```

Comment:
Is it possible that a bug in 2.53.3 could be found and a patch published with a fix? If so we'd have to change this again for folks to pick up the fix.